plugins/inputs/redfish/redfish.go

```diff
@@ -26,13 +26,21 @@ type Redfish struct {
 	Username         string          `toml:"username"`
 	Password         string          `toml:"password"`
 	ComputerSystemID string          `toml:"computer_system_id"`
+	IncludeTagSets   []string        `toml:"include_tag_sets"`
 	Timeout          config.Duration `toml:"timeout"`
 
+	TagSet map[string]bool
```

Comment:
Please do not export this field
```suggestion
	tagSet map[string]bool
```

Reply:
Ah rookie mistake!